apps/web/public/static/locales/en/common.json

```diff
@@ -1957,7 +1957,7 @@
   "members_default_location": "Member's default location",
   "members_default_schedule_description": "We will use each members default availability schedule. They will be able to edit or change it.",
   "requires_at_least_one_schedule": "You are required to have at least one schedule",
-  "default_conferencing_bulk_description": "Update the locations for the selected event types",
+  "default_conferencing_bulk_description": "Update the {{type}} for the selected event types",
```

Comment:
Using `default_conferencing_bulk_description` for schedules bulk update doesn't make sense.

Hey @SySagar, please keep this text as it is; no need to change it. Add a new text: `default_schedules_bulk_description: "Update the schedules for the selected event types"` as Amit suggested and pass it from the other two components apart from applist."